python/lsst/pipe/analysis/analysis.py

```diff
@@ -46,6 +46,7 @@ class AnalysisConfig(Config):
     doPlotRaDec = Field(dtype=bool, default=False, doc="Make delta vs. Ra and Dec plots?")
     doPlotFP = Field(dtype=bool, default=False, doc="Make FocalPlane plots?")
     doPlotCcdXy = Field(dtype=bool, default=False, doc="Make plots as a function of CCD x and y?")
+    doPlotTractOutline = Field(dtype=bool, default=True, doc="Plot tract outline (may be a bit slow)?")
```

Comment:
A "bit"? The comment below says "VERY". It's bad enough I suggest you should set it to `False` by default.